sycl/plugins/unified_runtime/ur/adapters/cuda/common.cpp

```diff
@@ -62,6 +62,29 @@ ur_result_t checkErrorUR(CUresult Result, const char *Function, int Line,
   throw mapErrorUR(Result);
 }
 
+ur_result_t checkErrorUR(ur_result_t Result, const char *Function, int Line,
```

Comment:
@kbenzie: I think we should have this method on common UR, so all adapters could use it.

Reply:
That seems reasonable, although this does have a CUDA specific environment variable check `std::getenv("PI_CUDA_ABORT")`. Perhaps this could be a follow up patch?

Reply:
Somewhat related, especially if a follow-up PR is going to happen, I've commented on the macro that wraps around `checkErrorUR` here: https://github.com/intel/llvm/pull/10244#discussion_r1255457150

Reply:
Thanks @kbenzie . Oh yes, follow-up patch is good. thanks @jchlanda for your reply as well.